testing/consul/APKBUILD

```diff
@@ -60,14 +36,18 @@ prepare() {
 }
 
 build() {
-	cd "$_builddir"/$pkgname || return 1
-	export GOPATH="$srcdir"
-	go build -v -o bin/consul \
-		-ldflags "-X main.GitDescribe $pkgver" || return 1
+	cd "$builddir"/$pkgname || return 1
+
+	mkdir -p "${builddir}/src/github.com/hashicorp"
+	ln -s "${builddir}" "${builddir}/src/github.com/hashicorp/consul"
+	cd "${builddir}/src/github.com/hashicorp/consul"
```

Comment:
Not really, the question was about if we wish to have `.../src/$pkgname-$pkgver/src/$host/$org/$pkgname` or `.../src/$pkgname-$pkgver` *and* `.../src/$host/$org/$pkgname`. But since you merged it, I guess it's a discussion for later, when we will come to the point where reality will make us regret not to have made that choice now. ;)